pkg/network/vmispec/interface.go

```diff
@@ -188,3 +188,19 @@ func HasBindingPluginDeviceInfo(iface v1.Interface, bindingPlugins map[string]v1
 	}
 	return false
 }
+
+// FilterInterfacesStatus returns a filtered slice of interface statues by some criteria
+// f must not be nil
+func FilterInterfacesStatus(
+	ifaceStatuses []v1.VirtualMachineInstanceNetworkInterface,
+	f func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool) []v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
I would call it `predicate` like the other filter we have.
But this is not a must.

Reply:
This code was removed.